File: src/main/java/com/dongtronic/diabot/data/nutrition/Photo.java

package com.dongtronic.diabot.data.nutrition;

import java.util.HashMap;
import java.util.Map;
import com.fasterxml.jackson.annotation.JsonAnyGetter;
import com.fasterxml.jackson.annotation.JsonAnySetter;
import com.fasterxml.jackson.annotation.JsonIgnore;
import com.fasterxml.jackson.annotation.JsonInclude;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonPropertyOrder;

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder({
    "thumb",
    "highres",
    "is_user_uploaded"
})
public class Photo {

    @JsonProperty("thumb")
    private String thumb;
    @JsonProperty("highres")
    private String highres;
    @JsonProperty("is_user_uploaded")
    private Boolean isUserUploaded;
    @JsonIgnore
    private Map<String, Object> additionalProperties = new HashMap<String, Object>();

    @JsonProperty("thumb")
    public String getThumb() {
        return thumb;
    }

    @JsonProperty("thumb")
    public void setThumb(String thumb) {
        this.thumb = thumb;
    }

    @JsonProperty("highres")
    public String getHighres() {
        return highres;
    }

    @JsonProperty("highres")
    public void setHighres(String highres) {
        this.highres = highres;
    }

    @JsonProperty("is_user_uploaded")
    public Boolean getIsUserUploaded() {
        return isUserUploaded;
    }

    @JsonProperty("is_user_uploaded")
    public void setIsUserUploaded(Boolean isUserUploaded) {
        this.isUserUploaded = isUserUploaded;
    }

    @JsonAnyGetter
    public Map<String, Object> getAdditionalProperties() {
        return this.additionalProperties;
    }

    @JsonAnySetter
    public void setAdditionalProperty(String name, Object value) {
        this.additionalProperties.put(name, value);
    }

}

File: src/main/java/com/dongtronic/diabot/data/nutrition/Photo.kt
package com.dongtronic.diabot.data.nutrition

import com.fasterxml.jackson.annotation.*
import java.util.*

@JsonInclude(JsonInclude.Include.NON_NULL)
@JsonPropertyOrder("thumb", "highres", "is_user_uploaded")
class Photo {
    @get:JsonProperty("thumb")
    @set:JsonProperty("thumb")
    @JsonProperty("thumb")
    var thumb: String? = null
    @get:JsonProperty("highres")
    @set:JsonProperty("highres")
    @JsonProperty("highres")
    var highres: String? = null
    @get:JsonProperty("is_user_uploaded")
    @set:JsonProperty("is_user_uploaded")
    @JsonProperty("is_user_uploaded")
    var isUserUploaded: Boolean? = null
    @JsonIgnore
    private val additionalProperties: MutableMap<String, Any> = HashMap()

    @JsonAnyGetter
    fun getAdditionalProperties(): Map<String, Any> {
        return additionalProperties
    }

    @JsonAnySetter
    fun setAdditionalProperty(name: String, value: Any) {
        additionalProperties[name] = value
    }
}